.github/workflows/integration.yaml

```diff
@@ -55,9 +54,8 @@ jobs:
        ACTIONS_ALLOW_UNSECURE_COMMANDS: true
      name: Python ${{ matrix.python-version }} ${{matrix.test-type}}-${{matrix.connection-type}} tests
      steps:
-       - uses: actions/checkout@v2
-       - name: install python
```

Comment:
same

Reply:
Not all of the `actions/setup-python` steps in the YAML files had a name beforehand, anyway.

I see no actual value in a longer YAML file with the only visible change being "install python" being shown instead of "Run actions/setup-python@v4" in logs.